src/redis_zset.cc

```diff
@@ -266,8 +266,17 @@ rocksdb::Status ZSet::RangeByScore(const Slice &user_key,
   rocksdb::Status s = GetMetadata(ns_key, &metadata);
   if (!s.ok()) return s.IsNotFound()? rocksdb::Status::OK():s;
 
+  // generate next possible score of max
+  int64_t i64 = 0;
+  double max_next_score = 0;
+  if (spec.reversed && !spec.maxex) {
+      memcpy(&i64, &spec.max, sizeof(spec.max));
```

Comment:
Can u help to add some comments about why it works by this week since that's a bit hacker to get the next score.

Reply:
OK. Comments will be submited right now